tests/smart_contracts/conftest.py

```diff
@@ -24,3 +41,80 @@ def dataset(organization: Organization) -> Dataset:
     )
 
     return dataset
+
+
+@pytest.fixture
+def agreements_dir() -> Path:
+    return Path(__file__).resolve().parent / "files" / "test_contracts"
+
+
+@pytest.fixture
+def signatures_dir() -> Path:
+    return Path(__file__).resolve().parent / "files" / "signatures"
+
+
+@pytest.fixture
+def agreement_one_signer(agreements_dir: Path) -> Path:
+    return agreements_dir / AGREEMENT_ONE_SIGNER
+
+
+@pytest.fixture
+def agreement_two_signers(agreements_dir: Path) -> Path:
+    return agreements_dir / AGREEMENT_TWO_SIGNERS
+
+
+@pytest.fixture
+def agreement_invalid(agreements_dir: Path) -> Path:
```

Comment:
A docstring should be added explaining why this file is considered invalid. That way, if someone later updates the logic and the file becomes valid, they won’t have to guess what the original definition of 'invalid' was.